### PR TITLE
Dr 0010 external file handling

### PR DESCRIPTION
**Pull request by clash99**
- Updated bullets on file structure for easier maintenance down the road

### CODE REVIEW

File: DR_0010_external-file-handling.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ The Cadasta Platform is inconsistent in its referencing of external static resou
 ## Suggested solution
 
 - Cadasta should adopt a standard practice of hosting all files locally on our servers.
-- These files should be located under `cadasta-platform/cadasta/core/static/` in their respective `css`, `fonts`, `img`, `js` folders.
+- New libraries or plug-ins should be located under `cadasta-platform/cadasta/core/static/`. 
+- A new folder with a descriptive name should be created.
+- A subfolder with the version number should be under the original folder.
```
Comment (oliverroick) [inline DR_0010_external-file-handling.md:18]: So we'd end up with a structure like `core/static/leaflet/1.0.4/{all the files here}`, right?

**Review by oliverroick — COMMENTED** [review]
One Question....

File: DR_0010_external-file-handling.md (hunk 1 above)
Comment (clash99) [reply DR_0010_external-file-handling.md:18]: Correct

Comment (oliverroick) [reply DR_0010_external-file-handling.md:18]: ok

**Review by oliverroick — APPROVED** [review]

**Review by amplifi — APPROVED** [review]

**Comment by seav** [comment]
Question: So where will our own static files be located? Do we create a "cadasta" directory under static?

**Pull request merged**